reports/Florida.md

```diff
@@ -71,6 +81,8 @@ id: fl-orlando-2
 
 Police officers are walking down a street. A woman in a purple shirt is pulled to the ground by several officers, who remain on top of her. Another woman yells at the officers to get off the first woman, and to get off her neck. The second woman is pepper-sprayed by an officer. They end up using tear-gas on the crowd.
 
+tags: tackle, shove, throw, kneel, kneel-on-kneck, pepper-spray, spray, tear-gas
```

Comment:
Should we go with kneel or knee?
or add kneel and kneel-on-neck to possible_tags.md?
also that's the second time I've seen kneck used... is that a thing... like kneck for a part of the human body?
no judgement, just wondering...  Corrected it to neck earlier

Reply:
Lol whoops that's my brain farting. I'll change to knee instead of kneel as well, thanks

Reply:
:smile: no worry, it's a lot of information...  not to mention the emotional impact